assets/locales/ru/translation.json

```diff
@@ -5,11 +5,13 @@
   "ja": "日本語",
   "zh-CN": "中文",
   "ko": "한국어",
-  "hi": "Hindi",
+  "hi": "Хинди",
   "kn": "Kannada",
   "ml-IN": "Malayalam",
-  "pa-IN": "Punjabi",
-  "te": "Telugu",
+  "pa-IN": "Панджаби",
```

Comment:
language names should be either not translated or should be spelled in their own languages

Reply:
yeah, i'm seeing the crowdin project